src-electron/main-process/modules/wallet-rpc.js

```diff
@@ -1292,30 +1292,38 @@ export class WalletRPC {
         params.payment_id = payment_id;
       }
 
-      this.sendRPC(rpc_endpoint, params).then(data => {
-        if (data.hasOwnProperty("error")) {
-          let error = data.error.message.charAt(0).toUpperCase() + data.error.message.slice(1);
+      this.sendRPC(rpc_endpoint, params)
+        .then(data => {
+          if (data.hasOwnProperty("error") || !data.hasOwnProperty("result")) {
+            let error = data.error.message.charAt(0).toUpperCase() + data.error.message.slice(1);
```

Comment:
`data.error` will be `undefined` if the first condition doesn't occur thus it'll crash if we try to access `message`